plugins/modules/lightsail.py

```diff
@@ -50,6 +50,22 @@
       - Launch script that can configure the instance with additional data.
     type: str
     default: ''
+  public_ports_info:
+    description:
+      - A list of dictionaries to describe the ports to open for the specified instance.
+    type: list
+    elements: dict
+    default:
+      - fromPort: 22
+        toPort: 22
+        protocol: 'tcp'
+        cidrs: ['0.0.0.0/0']
+        ipv6Cidrs: ['::/0']
+      - fromPort: 80
+        toPort: 80
+        protocol: 'tcp'
+        cidrs: ['0.0.0.0/0']
+        ipv6Cidrs: ['::/0']
```

Comment:
```suggestion
        ipv6Cidrs: ['::/0']
    version_added: 6.0.0
```

This on the other hand, is when this configuration was added.